src/components/MediaRecorder/AudioRecorder/AudioRecorder.tsx

```diff
@@ -19,47 +19,58 @@ export const AudioRecorder = () => {
 
   const isUploadingFile = recording?.$internal?.uploadState === 'uploading';
 
+  const state = useMemo(
+    () => ({
+      paused: recordingState === MediaRecordingState.PAUSED,
+      recording: recordingState === MediaRecordingState.RECORDING,
+      stopped: recordingState === MediaRecordingState.STOPPED,
+    }),
+    [recordingState],
+  );
```

Comment:
I think there is no need to memoize this. Inlining these values should be simpler:
```ts
const paused = recordingState === MediaRecordingState.PAUSED;
...
```